### PR TITLE
Improve Route53 documentation (#221)

### PR DESCRIPTION
**Pull request by geek1011**
- Grammar fixes
- Formatting improvements
- Don't encourage using environment vars instead of the credentials file

See #221

### CODE REVIEW

File: docs/_providers/route53.md [hunk 1]
```diff
@@ -24,13 +17,18 @@ It is also possible to specify an aws access key in the providers config json fi
 }
 {% endhighlight %}
 
-## Metadata
+You can also use environment variables, but this is discouraged.
```
Comment (tlimoncelli) [inline docs/_providers/route53.md]: Please add back this link `[go sdk configuration](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)`

Comment (geek1011) [reply docs/_providers/route53.md]: Ok, just curious, what is the point of it?

Comment (captncraig) [reply docs/_providers/route53.md]: Some users configure it in different ways. If you are running from certain instance types, the environment might be implicitly set up, or they want to manage aws creds in the same way they manage other tools.

Comment (geek1011) [reply docs/_providers/route53.md]: Ok, I see now. I don't use that method myself, so I did not see the point. I'll add it.

Comment (geek1011) [reply docs/_providers/route53.md]: @captncraig @tlimoncelli Done.

**Pull request merged**